server/controllers/availability.js

```diff
@@ -0,0 +1,88 @@
+const Availability = require("../models/Availability");
+const asyncHandler = require("express-async-handler");
+const mongoose = require("mongoose");
+
+// @route POST /availability
+// @desc create a schedule
+// @access private
+exports.createSchedule = asyncHandler(async (req, res, next) => {
+    const schedule = req.body.schedule
```

Comment:
It is a good practice to destructure the request body to only allow required fields

Reply:
not sure what you mean? do you mean just destructure the schedule variable like const { profileId, schedule } = req.body.schedule

Reply:
@natey37 You can ignore the above comment but provide additional context I've listed an example below:

```
const { startDate, endDate, sitterId } = req.body
```